### DISCUSSION TITLE
Driver chrashed 2 times on Daly BMS

### DISCUSSION THREAD

**Discussion by exenbema** [Louisvdw/dbus-serialbattery#230] [Q&A] [answered]
Hi big thx to this great work - i´m really be very thankful!

i have 2 indepentant systems running. 

1) ONE Daly BMS - Driver installed - works fine since 3 weeks
2) TWO Daly BMS - driver works fine but stopped 2 times 

Cerbo firmware 2.9 (and i tried 2,73)
2 Daly bms Firmware up2date: 2x11kwh lifepo4 (so 16x230AH each) 
3 Multiplus 48/5000 in parallel Mode

somewhere in the forum you said when you have multiple bms you should go back to cerbo 2.73 (or at least i think you said) but in the requirements: we need cerbo 2.80 - i tried to go to version 2.73 but then i could not install the driver. its said
root@einstein:~# sh installrelease.sh
mkdir: can't create directory '/opt/victronenergy/service-templates/dbus-serialbattery': No such file or directory
cp: omitting directory '/data/etc/dbus-serialbattery/service'
cp: can't create directory '/opt/victronenergy/service-templates/dbus-serialbattery/log': No such file or directory
cp: can't create '/opt/victronenergy/service-templates/dbus-serialbattery/run': No such file or directory

////////////

the thing is i cant read the logfiles, i dont get it, i already read the troubleshoot area...sorry

when the driver stopps the multipluses show low battery. and i saw in the discussions here that it might be a 9000Watt issue, so then the BMS say NONONO its to much, you wrote 180Amps but i think i have 2x115AMPs in total

can you help me? 
thx markus

**Comment by Louisvdw** [comment, marked as answer]
The latest driver (v0.12 or v013) only work on VenusOS 2.80 or newer (so use 2.90 or 2.91). 
The Daly does have some protocol issues and the driver is trying to work around those the best it can, but I cannot change the hardware limitations. But please try the new v0.13 it has some Daly fixes in.

**Reply by exenbema** [reply]
ok thx!! do you think its better to switch to JK? i could...

**Reply by exenbema** [reply]
i installed 2 days ago the driver - its a special routine to get the new driver? i will try yes thx

**Reply by Louisvdw** [reply]
If you have the Daly it should be fine. Just check the balancing of your cells often.
To install the new version is the same as installing the first time. But you already have the install script, so just run 
> sh /data/etc/dbus-serialbattery/installrelease.sh
